recipe/recipe.yaml

```diff
@@ -0,0 +1,62 @@
+schema_version: 1
+
+context:
+  min_python: python >=3.8,<3.13
```

Comment:
`min_python` is set here - is the lint checking for `python_min`?

Reply:
@conda-forge/help-python I need help fixing the noarch python linting issues in this recipe. 
I'm trying to convert from conda-build to rattler-build using feedrattler.

The linter is reporting:
- Missing lower bound on python version
- Issues with python version specification syntax

I've tried setting python_min but still getting errors. Any guidance would be appreciated!